www/include/Administration/parameters/knowledgeBase/formKnowledgeBase.php

```diff
@@ -33,14 +33,19 @@
  *
  */
 
+const PWS_OCCULTATION = '******';
```

Comment:
you'll probably have a notice cause this const is already defined in `www/include/Administration/parameters/general/form.php`